src/ServicePulse.Host/app/modules/configuration/js/notifications/emailnotifications.controller.js

```diff
@@ -0,0 +1,45 @@
+﻿(function (window, angular) {
+    'use strict';
+
+    function controller(
+        $scope,
+        $uibModalInstance,
+        notificationsService,
+        toastService
+    ) {
+        function refreshData() {
+            notificationsService.getSettings().then((notifications) => {
+                $scope.settings = notifications;
+            });
+        }
+
+        $scope.save = (notificationsForm) => {
+            if (notificationsForm.$valid) {
+                notificationsService.updateSettings($scope.settings).then(
+                    () => {
+                        toastService.showInfo('Eamil settings updated.');
+                        $uibModalInstance.dismiss('saved');
+                    },
+                    () => toastService.showError('Failed not update email settings.'));
```

Comment:
Is this yoda style ;)?

```suggestion
                    () => toastService.showError('Failed to update the email settings.'));
```